src/pages/SetPassword.vue

```diff
@@ -18,9 +22,19 @@
         name="password"
         type="password"
       />
-      <p>
-        Please choose a personal password to encrypt your new account.
-      </p>
+      <ae-label
+        :for="_uid"
+        :help-text="errors.first('password_repeat')"
+        help-type="dramatic"
+      >Confirm your password</ae-label>
+      <ae-input
+        v-validate="'required|confirmed:'+password"
```

Comment:
Should be possible to use field name instead of actual value.

Reply:
In custom elements, it doesn't know where to look for if field name is given.
https://github.com/baianat/vee-validate/issues/802
This suggests to use ref, but I can't get it to work.